### DISCUSSION TITLE
Start talking on Git

### DISCUSSION THREAD

**Discussion by EmmanuelSWE** [GeorgeMugale/Web-QR-Reaction-app#1] [General]
yo i hope only you guys can see this, lets get used to communicating on git

**Comment by GeorgeMugale** [comment]
My son i cannot see the new branch on my vs2022

**Reply by GeorgeMugale** [reply]
Oh nvm i see it, i just didn't fetch

**Comment by GeorgeMugale** [comment]
@Emmanuel222127212 Where are you

**Reply by EmmanuelSWE** [reply]
yo

**Reply by EmmanuelSWE** [reply]
must i change the location of the folders? i also added the css of the flash card template @GeorgeMugale

**Reply by GeorgeMugale** [reply]
I already changed the location
